docs/openapi3.yaml

```diff
@@ -1917,9 +1917,14 @@ components:
             Allows for the probability (in %) of a scenario being picked by a new virtual user to be "weighed" relative to other scenarios. If not specified, each scenario is equally likely to be picked.
           example: 20
     job:
+      discriminator:
+        propertyName: type
+      oneOf:
+          - $ref: '#/components/schemas/load_test'
+          - $ref: '#/components/schemas/functional_test'
       required:
+        - type
```

Comment:
breaking change in API,
in UI will include test type so won't break but all jobs created from API will break -> need to include in changelog